src/apps/Altinn.AccessManagement/src/Altinn.AccessManagement/Controllers/SystemUserClientDelegationController.cs

```diff
@@ -67,25 +67,20 @@ public async Task<ActionResult<ConnectionDto>> GetClientDelegations([FromQuery]
     /// </summary>
     /// <param name="party">The party the authenticated user is performing client administration on behalf of</param>
     /// <param name="systemUser">The system user the authenticated user is delegating access to</param>
-    /// <returns><seealso cref="ConnectionDto"/>List of connections</returns>
+    /// <returns><seealso cref="DelegationDto"/>List of connections</returns>
     [HttpGet]
     [Authorize(Policy = AuthzConstants.POLICY_CLIENTDELEGATION_READ)]
-    public async Task<ActionResult<ConnectionDto>> GetClientDelegations([FromQuery] Guid party, [FromQuery] Guid systemUser)
+    public async Task<ActionResult<DelegationDto>> GetClientDelegations([FromQuery] Guid party, [FromQuery] Guid systemUser)
     {
         var userId = AuthenticationHelper.GetPartyUuid(HttpContext);
         if (userId == Guid.Empty)
         {
             return Unauthorized();
         }
 
-        var dbResult = await connectionService.Get(fromId: null, toId: systemUser, facilitatorId: party);
-        var res = new List<ConnectionDto>();
-        foreach (var r in dbResult)
-        {
-            res.Add(ConnectionConverter.ConvertToDto(r));
-        }
-       
-        return Ok(res);
+        var result = await delegationService.Get(fromId: null, toId: systemUser, facilitatorId: party);
```

Comment:
Nøster delegationService fortsatt fra klienthovedenhet til underenheter?

Usikker på om det blir korrekt å bytte til delegation service. Ja, det er bare delegations man skal få ut her, men det må fortsatt nøste til underenheter om klienten som systembruker har mottatt delegering fra er en hovedenhet.

og, vi må også inn med nøstingen fra når klient er ENK til Innehaver som vi mangler enda (som for så vidt også er ett case vi glemte på workshop i går med frontend)